README.md

```diff
@@ -213,6 +220,90 @@ oneAgentSDK.addCustomRequestAttribute("salesAmount", 2500);
 
 When no service call is being traced, the custom request attributes are dropped.
 
+<a name="webrequests"/>
+
+## Trace web requests
+
+<a name="inwebrequests"/>
+
+### Trace incoming web requests
```

Comment:
same comment as before, please keep the format consistent with incoming and outgoing remote calls.